scripts/deploy_limited_allowed_recipients.py

```diff
@@ -0,0 +1,149 @@
+from brownie import chain, network
+
+from utils.config import (
```

Comment:
I suggest renaming the deploy script to smth like:
`deploy_limit_enabled_payout_factories`

Reply:
done